service/surf/apps/filters/utils.py

```diff
@@ -94,7 +94,7 @@ def _update_mptt_filter_category(filter_category, api_client):
 def _auto_enable_english_same_as_dutch(filter_item):
     translation = Locale.objects.create(
         asset=f"{filter_item.external_id}_auto_generated_at_{datetime.datetime.now().strftime('%c-%f')}",
-        en=filter_item.external_id, nl=filter_item.external_id, is_fuzzy=True
+        en=filter_item.external_id, nl=filter_item.external_id
```

Comment:
Ik dacht dat deze juist wel fuzzy moesten zijn? Omdat het niet altijd geldt?

Reply:
Voor de auteurs en publishers is het in ieder geval wel altijd hetzelfde. En daar willen ze dus niet naar kijken. Het enige is inderdaad dat voor dubbelen ofzo er een check moet plaatsvinden en dat zou je dan het beste tijdens het bekijken van vertalingen kunnen doen. Zal dat nog even voorleggen aan Jelmer.

Reply:
Het aanpakken van dubbelen wilt Jelmer in Sharekit of andere repo's doen. Dus een M. Janssen en Maria Janssen gaan we niet met elkaar vergelijken in Edusources. Dus het kan gelijk op niet-fuzzy.